content/post/why-should-we-trust-ml.md

```diff
@@ -0,0 +1,74 @@
+
+By applying LIME, we approximate a linear model for both the predictions of Hospital A and B. We recieve coefficeint estimates for our features around the local regions of these predictions.
+
+![](/img/lime_clinical.png)
+
+We can see that Hospital B has a higher predicted enrollment rate by our black box model and our approximate localized linear model. But why? With LIME, we can see that feature G is what contributes to the difference in enrollment rates. An analysis of that feature and how it differs between the two hospitals would help us understand why these enrollment rates are different, while also giving credit to the other positive features, and understanding that the number of trials in a city actually has a strong negative effect across both hospitals.
+
+We did not have to forego any predictive power to extract the interpretability we needed. It is important to note, however, that LIME gives us *local* interpretations, but if *global* interpretations are needed, some trade-off will need to occur.
+
+### SHAP
+
+SHAP values (short for SHapley Additive exPlanations) provides another framework for interpreting model predictions. It is based on Shapley values, a technique used in game theory to determine how much each player in a collaborative game has contributed to its success. In our case, each SHAP value measures how much each feature in our model contributes, either positively or negatively, to a model output. Its main benefits are that (1) it is completely model agnostic, so can be applied to tree based and black-box models and (2) each individual observation will have its own set of SHAP values, resulting in a local interpretation similar to LIME.
+
+![](/img/shap_explanation.png)
+
+The figure above estimates SHAP values for a particular observation, when adding in each covariate one by one. This diagram shows a single ordering, but when a model is non-linear and the features are not independent, the order of the features added to the expectation matters, and the SHAP values arise from averaging the \phi values across all possible orderings.
+
+A package to implement Shap values and the visualizations above is here: https://github.com/slundberg/shap 
+
+### Use Case: Mortality Risk Factors
+
+The NHEFS study was designed to investigate the relationships between clinical, nutritional, and behavioral factors and subsequent morbidity and mortality. This is a classic survival analysis problem, and can be studied through numerous interpretable linear modeling approaches. However, we want to build the most powerful model we can, accounting for non-linear patterns in the data, while also developing an intuition behind what is occurring. As such, a cox proportional hazards model was fit using XgBoost, to allow for more complex relationships between predictors.
```

Comment:
I think this post could be better if you picked one example that you used throughout -- you introduce it near the top, and then apply both SHAP and LIME to it. Having to introduce three different examples takes up a lot of text space and reader attention. One simple example I think will make the whole piece more direct and easier for the reader to understand